point_transformer_pytorch/point_transformer_pytorch.py

```diff
@@ -47,8 +47,8 @@ def forward(self, x, pos):
         v = v + rel_pos_emb
 
         # attention
-        attn = sim.softmax(dim = -1)
+        attn = sim.softmax(dim = -2)
```

Comment:
I cannot confirm which axis that the softmax conduct on.

Reply:
i'm pretty confident this is the right axis `j` - sim is `(batch x query rows x query-key similarities per row x feature dimension)`

Reply:
i'm just not sure if the attention MLP does not reduce to one attention map that aggregates across all feature value vectors. if someone else can confirm this is indeed the case, i'll merge and release the corrected version

Reply:
In my opinion, the point transformer in paper is used to informtion exchange or interaction, other than aggregation. The maxpooing in Pointnet++ is assigned to feature aggregation which aggregate many points feature to one point feature. In point transformer layer, the point number of outputs is same as the inputs.

Reply:
ok, i'll go by your word for now :)